src/util.ts

```diff
@@ -287,7 +286,14 @@ function removeManifestFromRegistry(manifestPath: string): Promise<any> {
     });
   }
   console.log(`Removing ${manifestPath}`);
-  return querySideloadingRegistry(['Remove-ItemProperty -LiteralPath $RegistryPath -Name "' + manifestPath + '" -ErrorAction SilentlyContinue']);
+  return new Promise(async (resolve, reject) => {
+    try {
+      const [parsedType, parsedGuid, parsedVersion] = await parseManifest(manifestPath);
+      resolve(await querySideloadingRegistry(['Remove-ItemProperty -LiteralPath $RegistryPath -Name "' + parsedGuid + '" -ErrorAction SilentlyContinue']));
```

Comment:
It would be better to use promisify so that resolve is an async method which you can await rather than mix async code with the promise code. There's an easy way to do this so you don't have to create the Promise yourself.